client/src/components/qa/addAnswer/AddAnsModal.jsx

```diff
@@ -2,7 +2,9 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import Modal from '../../shared/Modal';
 
-export default function AddAnsModal({ show, setShow, id }) {
+export default function AddAnsModal({
+  show, setShow, id, getAllAnswers,
```

Comment:
This is the same thing I mentioned above, but just to reiterate: don't pass down stateful functions